habit/src/components/TextInput.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { TextInput } from 'react-native'
+import PropTypes from 'prop-types';
+
+export default function CustomTextInput(props){
+    return  <TextInput 
```

Comment:
Let's use prettier, to style our code properly

Reply:
Also, share our prettier-rc files

Reply:
share, how?